docs/modules/project/pages/version-policy.adoc

```diff
@@ -0,0 +1,16 @@
+= gulp-asciidoctor Version Policy
+:keywords: release policy
+:uri-semver: https://semver.org/
+:uri-asciidoctorjs: https://asciidoctor-docs.netlify.app/asciidoctor.js/
+
+== Version Policy
+
+gulp-asciidoctor will follow the {uri-semver}[semantic versioning] rules. Each release will be versioned *major.minor.patch*.
+
+Major and Minor::
+Major and Minor release numbers will be the same as the underlying {uri-asciidcotrjs}[Asciidoctor.js] release numbers. That means, a gulp-asciidoctor release 2.3.x will be based on the Asciidoctor.js release 2.3.y.
```

Comment:
In theory it could work but in practice I don't think it's a good idea. We tried this approach with Asciidoctor (Ruby), Asciidoctor.js and AsciidoctorJ to sync the version number but it didn't work very well.

For instance, if gulp release a major version or if you want to do a breaking change (unrelated to Asciidoctor.js) then you cannot adhere to semantic versioning or you will have to wait Asciidoctor.js 3.0.

I think you should adhere to semantic versioning and state in the README which range of version is compatible. For instance, `gulp-asciidoctor` 3.x - 5.x is compatible with Asciidoctor.js 2 and `gulp-asciidoctor` 6.x is only compatible with Asciidoctor.js 3 or greater.

Reply:
Sound ok for me. I did not have such experiences but appreciate your!

I'll change it.